__tests__/app.test.js

```diff
@@ -154,4 +187,46 @@ describe("Error handling", () => {
         expect(response.body.msg).toBe("Not a valid id");
       });
   });
+  test('POST: 400 - Invalid ID (too big)', () => {
+    const comment = {username: `philippaclaire9`, body: `No comment`}
+    return request(app)
+    .post ("/api/reviews/6475/comments")
+    .send(comment)
+    .expect(400)
+    .then((response)=>{
+      expect(response.body.msg).toEqual("Invalid id")
+    })
+  });
+  test('POST: 400 ID provided is a string ', () => {
+    const comment = {username: `philippaclaire9`, body: `No comment`}
+    return request(app)
+    .post ("/api/reviews/boo/comments")
+    .send(comment)
+    .expect(400)
+    .then((response)=>{
+      expect(response.body.msg).toEqual("Invalid id")
+    })
+  });
+  test('POST: 400 missing body', () => {
+    const comment = {username: `philippaclaire9`, body: `   `}
```

Comment:
You should change this to {username: "philippaclaire9"} - whereby the body isn't present at all.